### ISSUE TITLE
Select box overflow bug

### ISSUE THREAD

**Issue by dmitrykrylov** [JiHong88/suneditor#1110]
**Describe the bug**
Incorrect select box submenu overflow behavior

**To Reproduce**
1. Put two editors next to each other with the "formats" plugin enabled
2. Click the "formats" button of the top editor
3. You will see that the select box popover is overflown by the second editor toolbar

Check the sandbox: https://codesandbox.io/s/suneditor-tooltip-overflow-bug-l4gyy4

**Expected behavior**
The toolbar shouldn't overflow over the select box popover

**Screenshots**
<img width="669" alt="Screen Shot 2022-10-19 at 12 20 16" src="https://user-images.githubusercontent.com/14343880/196652619-168a6ddb-b62e-4acb-8133-861f5b7199ab.png">

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Browser: Firefox
 - Version 106.0b9 (64-bit)

**Additional context**
The bug is most likely caused by [stacking contexts created](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context)

**Comment by JiHong88** [comment]
Try it.
```
suneditor.create('editor1',{
  defaultStyle: 'position:relative; z-index:1'
})
suneditor.create('editor2',{
  defaultStyle: 'position:relative; z-index:2'
})
```

**Comment by dmitrykrylov** [comment]
@JiHong88 this will fix the issue only for the one direction of the tooltip.

For instance, you proposed to set a higher z-index for the second editor. Then it will work well when the tooltip is positioned on the top (e.g. when the field is located on the bottom of the page):

<img width="792" alt="Screen Shot 2022-10-25 at 13 23 08" src="https://user-images.githubusercontent.com/14343880/197749141-26107214-0cb1-46cd-87b5-17df3ee0100d.png">

But it still will be overflown when the popover is positioned on the bottom:

<img width="790" alt="Screen Shot 2022-10-25 at 13 25 39" src="https://user-images.githubusercontent.com/14343880/197749614-842804dd-a2c2-45b1-811a-ba3f17147ab2.png">

And if you reorder z-indexes the opposite way, the opposite thing will happen.

**Comment by JiHong88** [comment]
In the current structure, it is difficult to improve this problem.
This has been fixed in version 3.0.0.
I will notify you after version update.
It will probably take several months..

**Comment by gitpash** [comment]
I can get that the root issue here is stacking context and proper solution (imo) is to render floating elements aside of the document flow (portal for react case for example)
personally I have several issues like this with `submenu` plugins like _fontSize_ and _fontColor_ for instance
at first, tried to modify a plugin code to add portal wrapper, but got many errors accessing some plugin's internals (this is my fault I think, maybe this is still a possible workaround, but needs more understandings of each particular plugin needs)
I ended up changing toolbar's css instead
```
    '.sun-editor .se-toolbar': {
        outline: 'none',
        borderBottom: THEME.border.base,
        zIndex: 'auto',
        // to be able to see tooltip on hover
        '& .se-tooltip-inner':  {
          zIndex: 999
        }
      }
```
this is not perfect, but works for me
